docs/release-notes/v13.1.0.md

```diff
@@ -0,0 +1,165 @@
+---
+id: v13.1.0
+title: Version 13.1.0 Release Notes
+sidebar_label: v13.1.0
+image: "img/release-note-link-preview.png"
+---
+
+_August 29th, 2024_
+
+:::info
+Mia-Platform Console v13.1.0 is **now in Preview** and will be generally available on September 12th.
+
+Console SaaS users can try out v13.1.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## Console
+
+### Breaking Changes

```

Comment:
This is not a breaking change yet, it will be breaking in the following minor.

Reply:
we should also warn users to upgrade their miactl version

Reply:
ok, checks the latest commits with the improvements

Reply:
The description seems good to me.

I'd add this suggestion to remove the _Breaking change_ title, since it is now unused.

```suggestion
```